### DISCUSSION TITLE
Some Packages Not Upgraded

### DISCUSSION THREAD

**Discussion by pengchengz30** [Homebrew/discussions#1626] [Casks]
I ran "brew upgrade" command, but some packages like "adobe-creative-cloud" and "visual-studio" they were not upgraded, unless I used "brew upgrade adobe-creative-cloud". Then it is upgrade. 
I was also not able to check which packages (not formulae) needed to be upgraded. The result of "brew outdated" is empty, which means those packages are not shown on the upgradable list.
Is it a bug and how can I solve it??
<img width="561" alt="Screenshot 2021-06-07 at 15 52 07" src="https://user-images.githubusercontent.com/65210538/120979580-52a65e00-c7a8-11eb-96f2-ee9a9c21b025.png">

**Comment by nandahkrishna** [comment]
These casks auto-update, which is why I presume Homebrew doesn't upgrade them when you run `brew upgrade`. You can use the `--greedy` option to override this behaviour.

See: https://github.com/Homebrew/homebrew-cask/blob/master/USAGE.md#updatingupgrading-casks

**Reply by pengchengz30** [reply]
That is the reason. Thanks!!
